### PR TITLE
ALD-779: Adding location for separate PHP files. Moving root from all…

### PR DESCRIPTION
**Pull request by sprymiker**
… location to server.

### CODE REVIEW

File: generator/src/templates/docker-compose.yml.twig [hunk 1]
```diff
@@ -88,7 +88,13 @@ services:
       ALLOWED_IP: "127.0.0.1"
     volumes:
       - ./${DEPLOYMENT_PATH}/context/nginx/conf.d/zed-rpc.default.conf:/etc/nginx/conf.d/default.conf:ro
-      - ./${DEPLOYMENT_PATH}/context/nginx/vhost.d:/etc/nginx/vhost.d:ro
+{% if _mountMode == 'baked' %}
```
Comment (sprymiker) [inline generator/src/templates/docker-compose.yml.twig:91]: @vlunov-spryker This is only for local docker env.

File: generator/src/templates/nginx/http/zed.server.conf.twig [hunk 2]
```diff
@@ -1,12 +1,2 @@
...
-{% endblock location %}
-{% block server %}
-{{ parent() }}
-
-    proxy_set_header Host $host;
-    proxy_set_header X-Forwarded-Proto $scheme;
-
-{% include "nginx/vhost.d/zed.default.conf.twig" with _context %}
-{% endblock server %}
+{% extends "nginx/http/server.conf.twig" %}
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/zed.server.conf.twig:1]: @vlunov-spryker This is only for local docker env.

File: generator/src/templates/nginx/http/application.server.conf.twig [hunk 3]
```diff
@@ -1,8 +1,20 @@
 {% extends "nginx/http/server.conf.twig" %}
 {% block locations %}
+    root /data/public/{{ endpointData['entry-point'] }};
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/application.server.conf.twig:3]: @vlunov-spryker Moving root from all locations to server. No impact.

File: generator/src/templates/nginx/http/application.server.conf.twig [hunk 4]
```diff
@@ -1,8 +1,20 @@
...
+    index index.php;
+
+    if ($request_uri ~* "^(.*/)index\.php(.*)$") {
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/application.server.conf.twig:6]: Always redirect index.php to / as Yves app show fail whale for index.php.

File: generator/src/templates/nginx/http/application.server.conf.twig [hunk 5]
```diff
@@ -1,8 +1,20 @@
...
+        return 301 $1;
+    }
 
 {% include "nginx/http/static.conf.twig" with _context %}
 
+    location ~ \.php$ {
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/application.server.conf.twig:12]: @vlunov-spryker Solving issues with downloading index and implementing version.php.

File: generator/src/templates/nginx/http/rpc.server.conf.twig [hunk 6]
```diff
@@ -1,4 +1,9 @@
 {% extends "nginx/http/application.server.conf.twig" %}
+{% block server %}
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/rpc.server.conf.twig:2]: @vlunov-spryker Adding missing zed server config.

File: generator/src/templates/nginx/http/static.server.conf.twig [hunk 7]
```diff
@@ -2,7 +2,7 @@
 {% block locations %}
 
     location / {
-        index index.html
+        index index.html;
```
Comment (sprymiker) [inline generator/src/templates/nginx/http/static.server.conf.twig:5]: Fixing syntax

File: generator/src/templates/nginx/http/application.server.conf.twig (hunk 3 above)
Comment (vlunov-spryker) [reply generator/src/templates/nginx/http/application.server.conf.twig:3]: Not relevant for our task.

File: generator/src/templates/docker-compose.yml.twig (hunk 1 above)
Comment (vlunov-spryker) [reply generator/src/templates/docker-compose.yml.twig:91]: OK.

File: generator/src/templates/nginx/http/static.server.conf.twig (hunk 7 above)
Comment (vlunov-spryker) [reply generator/src/templates/nginx/http/static.server.conf.twig:5]: OK.

File: generator/src/templates/nginx/http/rpc.server.conf.twig (hunk 6 above)
Comment (vlunov-spryker) [reply generator/src/templates/nginx/http/rpc.server.conf.twig:2]: If possible let us avoid.

**Pull request merged**